closure/goog/dom/forms_test.js

```diff
@@ -417,6 +417,26 @@ function testSetValueInput() {
   assertEquals('', goog.dom.forms.getValue(el));
 }
 
+function testGetValueMeter() {
+  var min = 0;
+  var max = 3;
+  var val = 2.3;
+  var el = goog.dom.createDom('meter', {'min': min, 'max': max, 'value': val});
+  assertEquals(val, goog.dom.forms.getValue(el));
+}
+
+function testSetValueMeter() {
+  var min = 1;
+  var max = 5;
+  var el = goog.dom.createDom('meter', {'min': min, 'max': max, 'value': 3});
+
+  assertEquals(3, goog.dom.forms.getValue(el));
+
+  var newValue = +(Math.random() * (max - min) + min).toPrecision(3);
```

Comment:
why would you avoid using random in tests? if anything min and max should be randomised too.